lib/eventboss/version.rb

```diff
@@ -1,3 +1,3 @@
 module Eventboss
-  VERSION = "1.4.1"
+  VERSION = "1.5.0"
```

Comment:
Oh really, why not `1.4.2`, it's just a small error notification addition 🤣

Reply:
Well, according to the SemVer manifesto: `PATCH version when you make backwards compatible bug fixes.`
But I'm fine with anything here - it's your toy in the end:).

Reply:
I agree with @tomaszkrzywda and remember we had similar discussion with @nglx with one of previous PRs 😄

Reply:
convinced.